src/main/java/com/puppycrawl/tools/checkstyle/checks/design/FinalClassCheck.java

```diff
@@ -42,6 +42,46 @@
  * &lt;module name=&quot;FinalClass&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * final class MyClass {
+ *   private MyClass() { } //OK
```

Comment:
```
 * final class MyClass { // OK
 *   private MyClass() { }
```

comment "ok" should be on a line where "violation" can occur, please add space after "//".
Please apply both rules to all code in your "Example:"

Reply:
Done